src/main/java/com/minecolonies/coremod/colony/README.md

```diff
@@ -1,77 +0,0 @@
-# About ColonyManager
```

Comment:
Sooo, i take it this all changed and isn't relevant now?

Reply:
I moved it to the package info